### PR TITLE
Add additional package for aufs support; fixes #74

### LINKED ISSUE #74: Missing recommended package for Ubuntu install of CS engine
The CS Docker engine Ubuntu docs show:

```
Install additional virtual drivers not in the base image.
$ sudo apt-get install -y linux-image-extra-virtual
```

There is a chance they will get devicemapper if the virtual package pulls down a newer version of the kernel that is available.  This puts the users in a non-supported configuration.  It should be:

```
Install additional virtual drivers not in the base image.
$ sudo apt-get install linux-image-extra-$(uname -r) linux-image-extra-virtual
```

/ping @mbentley

### PR DESCRIPTION
**Pull request by mbentley**
Signed-off-by: Matt Bentley matt.bentley@docker.com

### CODE REVIEW

**Comment by mbentley** [comment]
Also removed now unnecessary reboot due to the extra package; removed  the unsupported operating system documentation.

File: cs-engine/install.md [hunk 1]
```diff
@@ -101,33 +101,18 @@ to the `docker` group.
...
 
     ```bash
     $ echo "deb https://packages.docker.com/1.12/apt/repo ubuntu-trusty main" | sudo tee /etc/apt/sources.list.d/docker.list
     ```
 
-    This adds the repository of the latest version of CS Docker Engine for the
-    Ubuntu Trusty distribution. Change the "ubuntu-trusty" string to the
-    distribution you're using:
-
-    * debian-jessie (Debian 8)
```
Comment (thaJeztah) [inline cs-engine/install.md]: Is debian not supported at all?

Comment (joaofnfernandes) [inline cs-engine/install.md]: According to http://success.docker.com/Get_Help/Compatibility_Matrix, debian is not supported. @cpuguy83 or @mavenugo can you confirm this?

**Comment by joaofnfernandes** [comment]
LGTM from a docs perspective, waiting from feedback on whether CS engine supports Debian or not.

**Comment by thaJeztah** [comment]
ping @cpuguy83 ptal

**Comment by mbentley** [comment]
Whoops, accidentally deleted the branch before this was merged

**Comment by joaofnfernandes** [comment]
@cpuguy83 can you take a look?

**Comment by johndmulhausen** [comment]
Thanks to @cpuguy83 for the LGTM offline. :) Merging.

**Pull request merged**